res/mods.js

```diff
@@ -276,6 +276,14 @@ const mods = [
         "working": true,
         "versions": [ "1.8.9" ]
     },
+    {
+        "name": "TooManyItems",
+        "links": {
+            "github": "https://github.com/HowardZHY/TooManyItems-LegacyFabric/releases",
```

Comment:
```suggestion
            "github": "https://github.com/HowardZHY/TooManyItems-LegacyFabric",
```
Link to the main github repo for consistency with the other mods, otherwise goods good.